climpred/metrics.py

```diff
@@ -990,398 +1742,481 @@ def _msss_murphy(forecast, reference, dim=None, **metric_kwargs):
+            xmin = None
 
-    References:
-        * Griffies, S. M., and K. Bryan. “A Predictability Study of Simulated
-          North Atlantic Multidecadal Variability.” Climate Dynamics 13, no.
-          7–8 (August 1, 1997): 459–87. https://doi.org/10/ch4kc4.
+        if 'xmax' in metric_kwargs:
+            xmax = metric_kwargs['xmax']
+        else:
+            xmax = None
 
-    """
-    mae_skill = __mae.function(forecast, reference, dim=dim, **metric_kwargs)
-    std = reference.std(dim).mean()
-    if 'comparison' in metric_kwargs:
-        comparison = metric_kwargs['comparison']
-    else:
-        raise ValueError(
-            'Comparison needed to normalize NMSE. Not found in', metric_kwargs
-        )
-    fac = _get_norm_factor(comparison)
-    nmae_skill = mae_skill / std / fac
-    return nmae_skill
+        if 'tol' in metric_kwargs:
+            tol = metric_kwargs['tol']
+        else:
+            tol = 1e-6
+        ref_skill = _crps_quadrature(forecast, cdf_or_dist, xmin, xmax, tol)
+    forecast_skill = __crps.function(forecast, reference, **metric_kwargs)
+    skill_score = 1 - forecast_skill / ref_skill.mean('member')
+    return skill_score
 
 
-__nmae = Metric(
-    name='nmae',
-    function=_nmae,
-    positive=False,
-    probabilistic=False,
+__crpss = Metric(
+    name='crpss',
+    function=_crpss,
+    positive=True,
+    probabilistic=True,
     unit_power=0,
-    long_name='Normalized Mean Absolute Error',
-    minimum=0.0,
+    long_name='Continuous Ranked Probability Skill Score',
+    minimum=-np.inf,
     maximum=1.0,
-    perfect=0.0,
+    perfect=1.0,
 )
 
 
-def _uacc(forecast, reference, dim=None, **metric_kwargs):
-    """
-    Bushuk's unbiased ACC (uACC).
+def _crpss_es(forecast, reference, **metric_kwargs):
+    """Continuous Ranked Probability Skill Score Ensemble Spread.
 
-    .. math:: uACC = \\sqrt{PPP} = \\sqrt{MSSS}
+    # TODO: Unsure of what to put here.
 
-    Args:
-        * forecast (xr.object)
-        * reference (xr.object)
-        * dim (str): dimension to apply metric to
-        * comparison (str): name comparison needed for normalization factor `fac`, see
-            :py:func:`climpred.metrics._get_norm_factor`
-            (internally required to be added via **metric_kwargs)
-        * metric_kwargs: (optional) weights, skipna, see xskillscore.mse
+    .. math::
+        CRPSS = 1 - \\frac{CRPS(\\sigma^2_f)}{CRPS(\\sigma^2_o)}
 
+    Args:
+        forecast (xr.object): Forecast with ``member`` dim.
+        reference (xr.object): References without ``member`` dim.
+        weights (xarray object, optional): Weights to apply over dimension. Defaults to
+                                           ``None``.
+        skipna (bool, optional): If True, skip NaNs over dimension being applied to.
+                                 Defaults to ``False``.
 
-    Range:
-        * 1: perfect
-        * 0 - 1: better than climatology
+    # TODO: What is the min, max, etc. here?
```

Comment:
See here for CRPSS ES

Reply:
as I put:
 minimum=-np.inf,
    maximum=0.0,
    perfect=0.0,

Reply:
I think I was getting tired here. Not hard for me to look down! Thanks.